src/GitHubCodeReviewDashboard/GitHubCodeReviewDashboard/Pages/Index.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@page
+@model IndexModel
+@{
+    ViewData["Title"] = "Active Pull Requests";
+}
+
+@foreach (var (category, pullRequests) in (await Dashboard.GetCategorizedPullRequests()).OrderBy(i => i.Key))
+{
+    @if (string.IsNullOrEmpty(Model.Categories) ||Model.Categories.Split(",").Contains(category))
```

Comment:
```suggestion
    @if (string.IsNullOrEmpty(Model.Categories) || Model.Categories.Split(",").Contains(category))
```

Reply:
what is the above syntax.  why do we have:

```c#
@foreach (...
{
    @if
```

i.e. why is hte `if` in an `@` tag?

Reply:
That's the magic of .cshtml; the @ means it's C# code instead of regular HTML.

Reply:
isn't that already done given that you have `@foreach`.  it should be clear that it's C# code here, right?

Reply:
No, since that could also be HTML content being emitted for each entry.